test/types_test.py

```diff
@@ -26,15 +26,15 @@ def test_numpy(self, n_points=1000, n_trees=10):
             a = a.astype(random.choice([numpy.float64, numpy.float32, numpy.uint8, numpy.int16]))
             i.add_item(j, a)
 
-        i.build(n_trees)
+        i.build(n_trees, 1)
```

Comment:
see comment higher up on default argument – should be no reason to force the user to provide this argument by default

Reply:
Should be fixed now